docs/agent-api.asciidoc

```diff
@@ -7,22 +7,10 @@ endif::[]
 
 === `Agent` API
 
-NOTE: This is the API documentation for the Elastic APM Node.js Agent.
-For getting started,
-we recommend that you take a look at our framework specific documentation for either <<express,Express>>,
-<<hapi,hapi>>,
-<<koa,Koa>>,
-<<restify,Restify>>,
-<<fastify,Fastify>>,
-or <<custom-stack,custom frameworks>>.
-
-The Elastic APM agent for Node.js is a singleton.
-You get the agent instance by either requiring `elastic-apm-node` or `elastic-apm-node/start`.
-For details on the two approaches,
-see the <<advanced-setup,Setup and Configuration>> guide.
-
-The agent is also returned by the `start()` function,
-which allows you to require and start the agent on the same line:
+NOTE: This is API reference documentation for the Elastic APM Node.js Agent.
+For getting started, we recommend that you <<set-up,start here>>.
```

Comment:
What about something like this instead:

```
To get started, see <set up the agent>.
New to Elastic APM? <Start here>.
```

Reply:
Thoughts on just dropping the `NOTE: ...` completely?  The TOC sidebar should be sufficient to guide them to the top of the docs.

Reply:
Dropped in commit c9fb5c34

Reply:
++ Love it